storage/sendlog.go

```diff
@@ -1,14 +1,16 @@
 package storage
 
-import "github.com/mailbadger/app/entities"
+import (
+	"github.com/mailbadger/app/entities"
+)
 
 func (db *store) CreateSendLog(l *entities.SendLog) error {
 	return db.Create(l).Error
 }
 
-func (db *store) CountLogsByUUID(uuid string) (int, error) {
+func (db *store) CountLogsByUUID(uid string) (int, error) {
 	var count int
-	err := db.Model(&entities.SendLog{}).Where("uuid = ?", uuid).Count(&count).Error
+	err := db.Model(&entities.SendLog{}).Where("uid = ?", uid).Count(&count).Error
```

Comment:
The column is called `id` here right?
```suggestion
	err := db.Model(&entities.SendLog{}).Where("id = ?", uid).Count(&count).Error
```

Reply:
@FilipNikolovski yes.